sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/implementation/models/EventRoute.java

```diff
@@ -70,17 +74,6 @@ public String getFilter() {
         return this.filter;
     }
 
-    /**
-     * Set the filter property: An expression which describes the events which are routed to the endpoint.
-     *
-     * @param filter the filter value to set.
-     * @return the EventRoute object itself.
-     */
-    public EventRoute setFilter(String filter) {
```

Comment:
This field became mandatory in the latest swagger, so it is taken in at constructor time only